src/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -1,11 +1,8 @@
-import Issue, { Severity } from "../checking/issue";
-import { IssueCode } from "../checking/issueCodes";
+import Issue from "../issueHandling/issue";
+import vocabissue from "../issueHandling/knownIssues.json";
```

Comment:
```suggestion
import issueJson from "../issueHandling/knownIssues.json";
```
Needs to be renamed in the whole file.